.gitignore

```diff
@@ -6,3 +6,4 @@ test/build/
 self-coverage/
 *.swp
 needs-transpile.js
+/.idea
```

Comment:
```suggestion
```

I'd prefer not to add IDE specific excludes to the repository.  You can put this into `.git/info/exclude` on your own clone.